kotlin/browse-ogc-api-feature-service/src/main/java/com/esri/arcgisruntime/sample/browseogcapifeatureservice/MainActivity.kt

```diff
@@ -0,0 +1,246 @@
+     */
+    private fun loadLayer(selectedCollectionInfo: OgcFeatureCollectionInfo) {
+        // create the OGC feature collection table
+        val table = OgcFeatureCollectionTable(selectedCollectionInfo)
+
+        // populate the OGC feature collection table
+        val queryParameters = QueryParameters().apply {
+            maxFeatures = 1000
+        }
+
+        table.apply {
+            // set the feature request mode to manual (only manual is currently supported). In this mode,
+            // you must manually populate the table - panning and zooming won't request features
+            // automatically
+            featureRequestMode = ServiceFeatureTable.FeatureRequestMode.MANUAL_CACHE
+
+            // populate the table from the service
+            populateFromServiceAsync(queryParameters, false, null)
+
+            addDoneLoadingListener {
+                // create a feature layer from the OGC feature collection table
+                val ogcFeatureLayer = FeatureLayer(table).apply {
+                    // Choose a renderer for the layer based on the table.
+                    renderer = getRendererForTable(table) ?: this.renderer
+                }
+
+                mapView.map.operationalLayers.apply {
+                    // clear previous layers from the map
+                    clear()
+                    // add the layer to the map
+                    add(ogcFeatureLayer)
+                }
+
+                // zoom to the extent of the selected collection
+                val collectionExtent = selectedCollectionInfo.extent
+                if (!collectionExtent.isEmpty) {
+                    mapView.setViewpointGeometryAsync(collectionExtent, 100.0)
+                }
+
+                // hide the layer list
+                layerFAB.isExpanded = false
+            }
+        }
+    }
+
+    /**
+     * Return a simple renderer for points, lines and polygons.
+     */
+    private fun getRendererForTable(table: FeatureTable): Renderer? {
+        return when (table.geometryType) {
+            GeometryType.POINT, GeometryType.MULTIPOINT -> return SimpleRenderer(
+                SimpleMarkerSymbol(
+                    SimpleMarkerSymbol.Style.CIRCLE,
+                    Color.BLUE,
+                    5f
+                )
+            )
+            GeometryType.POLYGON, GeometryType.ENVELOPE -> return SimpleRenderer(
+                SimpleFillSymbol(
+                    SimpleFillSymbol.Style.SOLID,
+                    Color.BLUE,
+                    null
+                )
+            )
+            GeometryType.POLYLINE -> return SimpleRenderer(
+                SimpleLineSymbol(
+                    SimpleLineSymbol.Style.SOLID,
+                    Color.BLUE,
+                    1f
+                )
+            )
+            else -> null
+        }
+    }
+
+    /**
+     * Sets up UI behaviour. Closes expandable floating action button on touching the map view.
```

Comment:
UK spelling! Samples should use US English ;) 

```suggestion
     * Sets up UI behavior. Closes expandable floating action button on touching the map view.
```
```suggestion
     * Sets up UI behaviour. Closes expandable floating action button on touching the map view.
```